client/src/components/LayoutComponents/Home/index.jsx

```diff
@@ -1,19 +1,29 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import SignIn from '../../CommonComponents/SignIn/index';
 import { create } from 'jss';
 import rtl from 'jss-rtl';
-import { StylesProvider, jssPreset } from '@material-ui/core/styles';
+import {
+	StylesProvider,
+	jssPreset,
+	ThemeProvider,
+} from '@material-ui/core/styles';
+import NavBar from '../../CommonComponents/NavBar';
+import Theme from '../../../theme/MaterialUiTheme';
 
 // Configure JSS
 const jss = create({ plugins: [...jssPreset().plugins, rtl()] });
 
-function Home() {
+function Home(props) {
+	let { history } = props;
```

Comment:
You should only use `let` if you plan on reassigning the variable. But here you are not reassigning it, so use `const`